### ISSUE TITLE
Does this support family sharing?

### ISSUE THREAD

**Issue by stumpykilo** [PackagedCat/nativescript-iap#14]
Is there a specific `TransactionState` that needs to be added? It doesn't seem to work with `TransactionState.purchased` or `TransactionState.restored`.

**Comment by stumpykilo** [comment]
Actually, this **does** work! Please disregard.

**Comment by PackagedCat** [comment]
@stumpykilo Sorry for the late response. Thank you for reporting your results.

**Comment by stumpykilo** [comment]
@PackagedCat no worries. I do think there needs to be an added `REVOKED` state or status that can be read on restore or similar unless I'm missing something.

**Comment by PackagedCat** [comment]
@stumpykilo added "productsRevoked" event in 8.0.4 version for listen for revoked family-shared purchases. See [Readme](https://github.com/PackagedCat/nativescript-iap#ios-only-listen-for-revoked-family-shared-purchases)

**Comment by stumpykilo** [comment]
@PackagedCat thanks for the prompt reply and change! Do you know when 8.0.4 will be released?

**Comment by PackagedCat** [comment]
@stumpykilo the npm package has already been published.

**Comment by stumpykilo** [comment]
@PackagedCat thanks. I'm getting this error after updating and trying to run.
```
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'xcode'
Require stack:
- /Users/stumpykilo/src/myApp/node_modules/nativescript-iap/scripts/after-prepare.js
- /Users/stumpykilo/src/myApp/hooks/after-prepare/nativescript-iap.js
```

**Comment by stumpykilo** [comment]
Also, do you mind adding a small sample of how to interact with the data like you did for `purchaseUpdated`?

**Comment by PackagedCat** [comment]
> @PackagedCat thanks. I'm getting this error after updating and trying to run.
> 
> ```
> node:internal/modules/cjs/loader:936
>   throw err;
>   ^
> 
> Error: Cannot find module 'xcode'
> Require stack:
> - /Users/stumpykilo/src/myApp/node_modules/nativescript-iap/scripts/after-prepare.js
> - /Users/stumpykilo/src/myApp/hooks/after-prepare/nativescript-iap.js
> ```

fixed in 8.0.5

> Also, do you mind adding a small sample of how to interact with the data like you did for `purchaseUpdated`?

There's nothing special there. The event passes a list of product IDs (`data.productIds`) that have been revoked.

**Comment by stumpykilo** [comment]
@PackagedCat thanks. I'm getting another error when I update to `8.0.5`.
```
***** Fatal JavaScript exception - application has been terminated. *****
NativeScript encountered a fatal error: Uncaught ReferenceError: NativeClass is not defined
at
./node_modules/nativescript-iap/purchase/purchase.ios.js(file: node_modules/nativescript-iap/purchase/purchase.ios.js:85:0)
```

**Comment by PackagedCat** [comment]
@stumpykilo fixed in `8.0.7`
